x-pack/plugins/log_explorer/public/components/virtual_columns/column_tooltips/resource_column_tooltip.tsx

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { css } from '@emotion/react';
+import { EuiFlexGroup, EuiFlexItem, EuiText, EuiToken, useEuiTheme } from '@elastic/eui';
+import type { CustomGridColumnProps } from '@kbn/unified-data-table';
+import { resourceHeaderTooltipParagraph, resourceLabel } from '../../common/translations';
+import { HoverPopover } from '../../common/hover_popover';
+import { TooltipButtonComponent } from './tooltip_button';
+
+export const ResourceColumnTooltip = ({ column, headerRowHeight }: CustomGridColumnProps) => {
+  const { euiTheme } = useEuiTheme();
+  const spacingCSS = css`
+    margin-bottom: ${euiTheme.size.s};
+  `;
+
+  const spacingXsCss = css`
+    margin-bottom: ${euiTheme.size.xs};
```

Comment:
**nit**: can we move these styles definition and consume the theme tokens from the `euiThemeVars` dictionary? It's prevent this styles creation on each render.